spec/Vehicle/Vehicle.vspec

```diff
@@ -191,6 +191,12 @@ AmbientAirTemperature:
   unit: celsius
   description: Ambient air temperature outside the vehicle.
 
+IsFunctional:
+  datatype: boolean
+  type: sensor
+  description: Indicates if the vehicle is functional. A vehicle is considered functional if it can safely run at
```

Comment:
Looks like a catch all solution, which makes it less usable in my opinion. From my point of view the distinction between a vehicle being broken down (static object / hazard) and a vehicle having problems (slow vehicle) is an important one.

Reply:
No problem, as long as we can find a good definition of "broken down". Would something like:

_"Vehicle has serious problems and has stopped"_

... work @StefanWerder? I.e. a vehicle that has voluntarily stopped/parked at a bad location is not "BrokenDown", and a vehicle being towed or driving very slowly due to a problem is also not "BrokenDown"

Reply:
We could of course try to promote based on the Sensoris definition: "Vehicle breakdown of ego vehicle or any similar event of ego vehicle stopped on the road, that might pose a risk to other road users.".

But I assume that would be a quite difficult for a vehicle to calculate, if it must estimate that there is a risk to other road users involved. If e.g. a vehicle has a flat tire or engine problems, but is correctly parked (i.e. no risk involved), shall then still `Vehicle.BrokenDown` be true? And in the opposite case - the vehicle is fully functional put parked where it might pose a risk to other road users.

But that we could maybe live if we add a comment like:

`comment: Actual criteria and method used to decide if a vehicle is broken down is implementation specific`

Reply:
I like the comment with it being implementation specific :)

Perhaps you can re-check the usage of the signal for our application inside Bosch.